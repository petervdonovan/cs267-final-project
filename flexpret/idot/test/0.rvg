[mu []
[[file {Reset_Handler} {
  Reset_Handler:
    [get-coreid t2]
    [critical-hart a1 a3 [sim-print t2 t1]]
    [single a1 [sim-print-int 42 t0 t1]]
    [single-hart a5 [critical-cores-mod8 a0 a6 [sim-print-int 12 t0 t1]]]
    [id die]}]]
]
