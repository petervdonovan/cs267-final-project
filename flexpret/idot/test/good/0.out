Core-0:          0
Core-1:          1
Core-2:          2
Core-3:          3
Core-4:          4
Core-5:          5
Core-6:          6
Core-7:          7
Core-8:          8
Core-9:          9
Core-10:         10
Core-11:         11
Core-12:         12
Core-13:         13
Core-14:         14
Core-15:         15
Core-16:         16
Core-17:         17
Core-18:         18
Core-19:         19
Core-20:         20
Core-21:         21
Core-22:         22
Core-23:         23
Core-24:         24
Core-25:         25
Core-26:         26
Core-27:         27
Core-28:         28
Core-29:         29
Core-30:         30
Core-31:         31
Core-32:         32
Core-33:         33
Core-34:         34
Core-35:         35
Core-36:         36
Core-37:         37
Core-38:         38
Core-39:         39
Core-40:         40
Core-41:         41
Core-42:         42
Core-43:         43
Core-44:         44
Core-45:         45
Core-46:         46
Core-47:         47
Core-48:         48
Core-49:         49
Core-50:         50
Core-51:         51
Core-52:         52
Core-53:         53
Core-54:         54
Core-55:         55
Core-56:         56
Core-57:         57
Core-58:         58
Core-59:         59
Core-60:         60
Core-61:         61
Core-62:         62
Core-63:         63
Core-0:          0
Core-1:          1
Core-2:          2
Core-3:          3
Core-4:          4
Core-5:          5
Core-6:          6
Core-7:          7
Core-8:          8
Core-9:          9
Core-10:         10
Core-11:         11
Core-12:         12
Core-13:         13
Core-14:         14
Core-15:         15
Core-16:         16
Core-17:         17
Core-18:         18
Core-19:         19
Core-20:         20
Core-21:         21
Core-22:         22
Core-23:         23
Core-24:         24
Core-25:         25
Core-26:         26
Core-27:         27
Core-28:         28
Core-29:         29
Core-30:         30
Core-31:         31
Core-32:         32
Core-33:         33
Core-34:         34
Core-35:         35
Core-36:         36
Core-37:         37
Core-38:         38
Core-39:         39
Core-40:         40
Core-41:         41
Core-42:         42
Core-43:         43
Core-44:         44
Core-45:         45
Core-46:         46
Core-47:         47
Core-48:         48
Core-49:         49
Core-50:         50
Core-51:         51
Core-52:         52
Core-53:         53
Core-54:         54
Core-55:         55
Core-56:         56
Core-57:         57
Core-58:         58
Core-59:         59
Core-60:         60
Core-61:         61
Core-62:         62
Core-63:         63
Core-0:          0
Core-1:          1
Core-2:          2
Core-3:          3
Core-4:          4
Core-5:          5
Core-6:          6
Core-7:          7
Core-8:          8
Core-9:          9
Core-10:         10
Core-11:         11
Core-12:         12
Core-13:         13
Core-14:         14
Core-15:         15
Core-16:         16
Core-17:         17
Core-18:         18
Core-19:         19
Core-20:         20
Core-21:         21
Core-22:         22
Core-23:         23
Core-24:         24
Core-25:         25
Core-26:         26
Core-27:         27
Core-28:         28
Core-29:         29
Core-30:         30
Core-31:         31
Core-32:         32
Core-33:         33
Core-34:         34
Core-35:         35
Core-36:         36
Core-37:         37
Core-38:         38
Core-39:         39
Core-40:         40
Core-41:         41
Core-42:         42
Core-43:         43
Core-44:         44
Core-45:         45
Core-46:         46
Core-47:         47
Core-48:         48
Core-49:         49
Core-50:         50
Core-51:         51
Core-52:         52
Core-53:         53
Core-54:         54
Core-55:         55
Core-56:         56
Core-57:         57
Core-58:         58
Core-59:         59
Core-60:         60
Core-61:         61
Core-62:         62
Core-63:         63
Core-0:          0
Core-1:          1
Core-2:          2
Core-3:          3
Core-4:          4
Core-5:          5
Core-6:          6
Core-7:          7
Core-8:          8
Core-9:          9
Core-10:         10
Core-11:         11
Core-12:         12
Core-13:         13
Core-14:         14
Core-15:         15
Core-16:         16
Core-17:         17
Core-18:         18
Core-19:         19
Core-20:         20
Core-21:         21
Core-22:         22
Core-23:         23
Core-24:         24
Core-25:         25
Core-26:         26
Core-27:         27
Core-28:         28
Core-29:         29
Core-30:         30
Core-31:         31
Core-32:         32
Core-33:         33
Core-34:         34
Core-35:         35
Core-36:         36
Core-37:         37
Core-38:         38
Core-39:         39
Core-40:         40
Core-41:         41
Core-42:         42
Core-43:         43
Core-44:         44
Core-45:         45
Core-46:         46
Core-47:         47
Core-48:         48
Core-49:         49
Core-50:         50
Core-51:         51
Core-52:         52
Core-53:         53
Core-54:         54
Core-55:         55
Core-56:         56
Core-57:         57
Core-58:         58
Core-59:         59
Core-60:         60
Core-61:         61
Core-62:         62
Core-63:         63
Core-0:          0
Core-1:          1
Core-2:          2
Core-3:          3
Core-4:          4
Core-5:          5
Core-6:          6
Core-7:          7
Core-8:          8
Core-9:          9
Core-10:         10
Core-11:         11
Core-12:         12
Core-13:         13
Core-14:         14
Core-15:         15
Core-16:         16
Core-17:         17
Core-18:         18
Core-19:         19
Core-20:         20
Core-21:         21
Core-22:         22
Core-23:         23
Core-24:         24
Core-25:         25
Core-26:         26
Core-27:         27
Core-28:         28
Core-29:         29
Core-30:         30
Core-31:         31
Core-32:         32
Core-33:         33
Core-34:         34
Core-35:         35
Core-36:         36
Core-37:         37
Core-38:         38
Core-39:         39
Core-40:         40
Core-41:         41
Core-42:         42
Core-43:         43
Core-44:         44
Core-45:         45
Core-46:         46
Core-47:         47
Core-48:         48
Core-49:         49
Core-50:         50
Core-51:         51
Core-52:         52
Core-53:         53
Core-54:         54
Core-55:         55
Core-56:         56
Core-57:         57
Core-58:         58
Core-59:         59
Core-60:         60
Core-61:         61
Core-62:         62
Core-63:         63
Core-0:          0
Core-1:          1
Core-2:          2
Core-3:          3
Core-4:          4
Core-5:          5
Core-6:          6
Core-7:          7
Core-8:          8
Core-9:          9
Core-10:         10
Core-11:         11
Core-12:         12
Core-13:         13
Core-14:         14
Core-15:         15
Core-16:         16
Core-17:         17
Core-18:         18
Core-19:         19
Core-20:         20
Core-21:         21
Core-22:         22
Core-23:         23
Core-24:         24
Core-25:         25
Core-26:         26
Core-27:         27
Core-28:         28
Core-29:         29
Core-30:         30
Core-31:         31
Core-32:         32
Core-33:         33
Core-34:         34
Core-35:         35
Core-36:         36
Core-37:         37
Core-38:         38
Core-39:         39
Core-40:         40
Core-41:         41
Core-42:         42
Core-43:         43
Core-44:         44
Core-45:         45
Core-46:         46
Core-47:         47
Core-48:         48
Core-49:         49
Core-50:         50
Core-51:         51
Core-52:         52
Core-53:         53
Core-54:         54
Core-55:         55
Core-56:         56
Core-57:         57
Core-58:         58
Core-59:         59
Core-60:         60
Core-61:         61
Core-62:         62
Core-63:         63
Core-0:          0
Core-1:          1
Core-2:          2
Core-3:          3
Core-4:          4
Core-5:          5
Core-6:          6
Core-7:          7
Core-8:          8
Core-9:          9
Core-10:         10
Core-11:         11
Core-12:         12
Core-13:         13
Core-14:         14
Core-15:         15
Core-16:         16
Core-17:         17
Core-18:         18
Core-19:         19
Core-20:         20
Core-21:         21
Core-22:         22
Core-23:         23
Core-24:         24
Core-25:         25
Core-26:         26
Core-27:         27
Core-28:         28
Core-29:         29
Core-30:         30
Core-31:         31
Core-32:         32
Core-33:         33
Core-34:         34
Core-35:         35
Core-36:         36
Core-37:         37
Core-38:         38
Core-39:         39
Core-40:         40
Core-41:         41
Core-42:         42
Core-43:         43
Core-44:         44
Core-45:         45
Core-46:         46
Core-47:         47
Core-48:         48
Core-49:         49
Core-50:         50
Core-51:         51
Core-52:         52
Core-53:         53
Core-54:         54
Core-55:         55
Core-56:         56
Core-57:         57
Core-58:         58
Core-59:         59
Core-60:         60
Core-61:         61
Core-62:         62
Core-63:         63
Core-0:          0
Core-1:          1
Core-2:          2
Core-3:          3
Core-4:          4
Core-5:          5
Core-6:          6
Core-7:          7
Core-8:          8
Core-9:          9
Core-10:         10
Core-11:         11
Core-12:         12
Core-13:         13
Core-14:         14
Core-15:         15
Core-16:         16
Core-17:         17
Core-18:         18
Core-19:         19
Core-20:         20
Core-21:         21
Core-22:         22
Core-23:         23
Core-24:         24
Core-25:         25
Core-26:         26
Core-27:         27
Core-28:         28
Core-29:         29
Core-30:         30
Core-31:         31
Core-32:         32
Core-33:         33
Core-34:         34
Core-35:         35
Core-36:         36
Core-37:         37
Core-38:         38
Core-39:         39
Core-40:         40
Core-41:         41
Core-42:         42
Core-43:         43
Core-44:         44
Core-45:         45
Core-46:         46
Core-47:         47
Core-48:         48
Core-49:         49
Core-50:         50
Core-51:         51
Core-52:         52
Core-53:         53
Core-54:         54
Core-55:         55
Core-56:         56
Core-57:         57
Core-58:         58
Core-59:         59
Core-60:         60
Core-61:         61
Core-62:         62
Core-63:         63
Core-0:         42
Core-0:         12
Core-8:         12
Core-16:         12
Core-24:         12
Core-32:         12
Core-40:         12
Core-48:         12
Core-56:         12
Core-1:         12
Core-9:         12
Core-17:         12
Core-25:         12
Core-33:         12
Core-41:         12
Core-49:         12
Core-57:         12
Core-2:         12
Core-10:         12
Core-18:         12
Core-26:         12
Core-34:         12
Core-42:         12
Core-50:         12
Core-58:         12
Core-3:         12
Core-11:         12
Core-19:         12
Core-27:         12
Core-35:         12
Core-43:         12
Core-51:         12
Core-59:         12
Core-4:         12
Core-12:         12
Core-20:         12
Core-28:         12
Core-36:         12
Core-44:         12
Core-52:         12
Core-60:         12
Core-5:         12
Core-13:         12
Core-21:         12
Core-29:         12
Core-37:         12
Core-45:         12
Core-53:         12
Core-61:         12
Core-6:         12
Core-14:         12
Core-22:         12
Core-30:         12
Core-38:         12
Core-46:         12
Core-54:         12
Core-62:         12
Core-7:         12
Core-15:         12
Core-23:         12
Core-31:         12
Core-39:         12
Core-47:         12
Core-55:         12
Core-63:         12
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
Core-16 is done
Core-17 is done
Core-18 is done
Core-19 is done
Core-20 is done
Core-21 is done
Core-22 is done
Core-23 is done
Core-24 is done
Core-25 is done
Core-26 is done
Core-27 is done
Core-28 is done
Core-29 is done
Core-30 is done
Core-31 is done
Core-32 is done
Core-33 is done
Core-34 is done
Core-35 is done
Core-36 is done
Core-37 is done
Core-38 is done
Core-39 is done
Core-40 is done
Core-41 is done
Core-42 is done
Core-43 is done
Core-44 is done
Core-45 is done
Core-46 is done
Core-47 is done
Core-48 is done
Core-49 is done
Core-50 is done
Core-51 is done
Core-52 is done
Core-53 is done
Core-54 is done
Core-55 is done
Core-56 is done
Core-57 is done
Core-58 is done
Core-59 is done
Core-60 is done
Core-61 is done
Core-62 is done
Core-63 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
    at Top.scala:126 assert(false.B, "Program terminated sucessfully")
[6046] %Error: Top.sim.v:1944776: Assertion failed in TOP.Top
%Error: Top.sim.v:1944776: Verilog $stop
Aborting...
